source/core/src/main/com/csse3200/game/GdxGame.java

```diff
@@ -72,13 +74,15 @@ private Screen newScreen(ScreenType screenType) {
         return new MainGameScreen(this);
       case SETTINGS:
         return new SettingsScreen(this);
```

Comment:
Not specific to this branch, but probably more maintainable in future to replace this with enhanced switch statement (and keep default for readability).